client/components/popover/index.jsx

```diff
@@ -71,89 +71,101 @@ var Popover = React.createClass( {
 		return null;
 	},
 
-	_showOrHideTip: function( prevProps ) {
-		if ( ! this._tip ) {
-			this._tip = new Tip( '' );
-			this._tip.classname = this.props.className;
-			this._container = this._tip.inner;
+	_initTip: function() {
+		this._tip = new Tip( '' );
+		this._tip.classname = this.props.className;
+		this._container = this._tip.inner;
+	},
+
+	_renderTip: function() {
+		let content = <Content { ...omit( this.props, 'className' ) } onClickOutside={ this._onClickOutside } onClose={ this._close } />;
+
+		// Context is lost when creating a new render hierarchy, so ensure
+		// that we preserve the context that we care about
+		if ( this.context.store ) {
+			content = (
+				<ReduxProvider store={ this.context.store }>
+					{ content }
+				</ReduxProvider>
+			);
 		}
 
-		if ( this.props.isVisible && this.props.context ) {
-			let content = <Content { ...omit( this.props, 'className' ) } onClose={ this._close } />;
-
-			// Context is lost when creating a new render hierarchy, so ensure
-			// that we preserve the context that we care about
-			if ( this.context.store ) {
-				content = (
-					<ReduxProvider store={ this.context.store }>
-						{ content }
-					</ReduxProvider>
+		// this schedules a render, but does not actually render the content
+		ReactDom.render( content, this._container );
+	},
+
+	_showTip: function() {
+		// defer showing the content to give it a chance to render
+		defer( () => {
+			const contextNode = ReactDom.findDOMNode( this.props.context );
+			if ( contextNode.nodeType !== Node.ELEMENT_NODE || contextNode.nodeName.toLowerCase() === 'svg' ) {
+				warn(
+					'Popover is attached to a %s element (nodeType %d).  ' +
+					'This causes problems in IE11 - see 12168-gh-calypso-pre-oss.',
```

Comment:
Not really, I doubt anything will parse the comment, so it only needs to have a good enough reference for people.